interface/patient_file/encounter/trend_form.php

```diff
@@ -45,51 +56,50 @@
   .graph {color:#0000cc;}
 
   #chart {
-    border-style:solid;
-    border-width:2px;
     margin:0em 1em 2em 2em;
```

Comment:
didn't want to mess with the spacing since it's already a bit crowded with the new graphing (the border was super ugly though, so removed it).